doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+TSPC_MESH_12_2	True		GATT Server (C.2)
+TSPC_MESH_12_3	True		Advertising with Network ID (C.2)
+TSPC_MESH_12_4	True		Advertising with Node Identity (C.2)
+TSPC_MESH_12_5	False (*)	Proxy Client (C.3)
+TSPC_MESH_12_6	False (*)	GATT Client (C.4)
+==============	==============	=======================================
+
+
+Mesh GATT Services
+==================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_13_1	True		Mesh Provisioning Service (C.1)
+TSPC_MESH_13_2	True		Mesh Proxy Service (C.2)
+==============	==============	=======================================
+
+
+GATT Server Requirements
+========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_14_1	True		Discover All Primary Services (M)
+TSPC_MESH_14_2	True		Discover Primary Services by Service UUID (M)
+TSPC_MESH_14_3	True		Write without Response (M)
+TSPC_MESH_14_4	True		Notification (M)
+TSPC_MESH_14_5	True		Write Characteristic Descriptors (M)
+==============	==============	=======================================
+
+
+GATT Client Requirements
+========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_15_1	False (*)	Discover All Primary Services (C.1)
+TSPC_MESH_15_2	False (*)	Discover Primary Services by Service UUID (C.1)
+TSPC_MESH_15_3	False (*)	Write without Response (M)
+TSPC_MESH_15_4	False (*)	Notification (M)
+TSPC_MESH_15_5	False (*)	Write Characteristic Descriptors (M)
+==============	==============	=======================================
+
+
+GAP Requirements
+================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_16_1	True		Broadcaster (C.1)
+TSPC_MESH_16_2	True		Observer (C.1)
+TSPC_MESH_16_3	True		Peripheral (C.2)
+TSPC_MESH_16_4	True		Peripheral - Security Mode 1 (C.2)
+TSPC_MESH_16_5	False (*)	Central (C.3)
+TSPC_MESH_16_6	False (*)	Central - Security Mode 1 (C.3)
+==============	==============	=======================================
+
+
+Provisioner - Bearers
+=====================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_17_1	False (*)	Advertising Bearer (C.1)
+TSPC_MESH_17_2	False (*)	GATT Bearer (C.1)
+==============	==============	=======================================
+
+
+Provisioner - Provisioning
+==========================
+
+===============	==============	=======================================
```

Comment:
I'm a bit annoyed by mixing tabs and spaces in these tables (it works but looks odd), but I'll still approve the PR.